### ISSUE TITLE
Removing Tag in v2.3.1 drops Error 404 "not found"

### ISSUE THREAD

**Issue by FiSHYsolutions** [wallabag/wallabag#3559]
### Issue details

Trying to remove a tag from a bookmark, results in new page:

> Oops! An Error Occurred
> The server returned a "404 Not Found".
> Something is broken. Please let us know what you were doing when this error occurred. We will fix it as soon as possible. Sorry for any inconvenience caused.

And the tag is not removed.

### Environment

* wallabag v2.3.1
* shared hosting environment
* setup: `wget -q https://wllbg.org/latest-v2-package && tar -xzf latest-v2-package`
* config: mostly default, only necessary changes
* PHP v7.1.11
* MySQL v5.7.20
* Ubuntu 16.04.3 LTS

### Steps to reproduce/test case

Try to delete a tag from a bookmark.

**Comment by j0k3r** [comment]
Can you share the `var/logs/prod.log`?
What's the name of the tag?

**Comment by FiSHYsolutions** [comment]
```
bash$ cat prod.log
[2018-01-16 08:05:41] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /remove-tag//29" (from "http://sub.domain.tld/view/47")" at /www/htdocs/user/domainfolder/2.3.1/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/EventListener/RouterListener.php line 123 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /remove-tag//29\" (from \"http://sub.domain.tld/view/47\") at /www/htdocs/user/domainfolder/2.3.1/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/EventListener/RouterListener.php:123, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /www/htdocs/user/domainfolder/2.3.1/var/cache/prod/appProdProjectContainerUrlMatcher.php:1012)"} []
```

**Comment by FiSHYsolutions** [comment]
The **tag contains blanks**: `alphanumstringa alphanumstringb alphanumstringc`

That was not intended, i forgot to write commas to seperate the strings. after saving and creation the blank-containing-tag i tried to remove it immediately, because of the blanks (i need 3 seperate tags). everything works fine, except the removement.

**Comment by j0k3r** [comment]
I got it.
The bug has been fixed already: https://github.com/wallabag/wallabag/issues/3534
It'll be available in the 2.3.2 release.

Meanwhile, if you want to remove the tag from the entry, you can directly call `http://sub.domain.tld/remove-tag/47/29`

**Comment by FiSHYsolutions** [comment]
workaround worked.
thank you very much.
